float32[] azimuth
float32[] elevation
float32[] radius
float32[] confidence
float64[] timestamp
rosMsgs/s_camera_type[] cameraType
float32[] relativeCameraX
float32[] relativeCameraY
float32[] relativeCameraZ
float32[] relativeCameraPhi
---
